### DISCUSSION TITLE
How to show some hint/helper/popover text for button action ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#14233] [Help] [answered]
### Package

Actions

### Package Version

^3.2-stable

### How can we help you?

Reading at https://filamentphp.com/docs/3.x/tables/actions how actions work I search how to show some hint/helper/popover text for any button action? Are there something like that ?

**Comment by leandrocfe** [comment, marked as answer]
what about `tooltip`?
